### PR TITLE
Update viewing-jit-dumps.md

### PR DESCRIPTION
**Pull request by EgorBo**
Updated some versions to the latest,
Added powershell example
Added `/y` parameter to `copy`, otherwise it asks for confirmation but nobody sees it because of `> NUL`

### CODE REVIEW

File: Documentation/building/viewing-jit-dumps.md [hunk 1]
```diff
@@ -107,6 +107,9 @@ These can be set in one of three ways:
    # Windows
    set COMPlus_JitDump=Main
 
+   # Powershell
+   $env:COMPlus_JitDump="Main"
```
Comment (fiigii) [inline Documentation/building/viewing-jit-dumps.md]: 👍

**Comment by gfoidl** [comment]
@EgorBo :wink: we have quite similar traces around -> https://github.com/dotnet/coreclr/pull/21687

**Comment by EgorBo** [comment]
@gfoidl oh, can you add /y there? and the powershell example if you think it's needed (I do) so I can close this

**Comment by gfoidl** [comment]
@EgorBo done. Thx.

**Pull request closed without merging**